tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
```

Comment:
@MiLk What do you prefer ? As long as we don't need something else than `exim-daemon-light`, I would prefer it being hard-coded, but I don't mind making it a variable to be honest. See together what you prefer, tell me and I'll do ;-)